third_party/terraform/tests/resource_storage_hmac_key_test.go

```diff
@@ -0,0 +1,56 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccStorageHmacKey_update(t *testing.T) {
+	t.Parallel()
+
+	saName := fmt.Sprintf("%v%v", "service-account", acctest.RandString(10))
+	bucketName := testBucketName()
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckStorageHmacKeyDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccGoogleStorageHmacKeyBasic(saName, bucketName, "ACTIVE"),
+			},
+			{
+				ResourceName:      "google_storage_hmac_key.key",
+				ImportState:       true,
+				ImportStateVerify: true,
```

Comment:
do I need the `ImportStateVerifyIgnore: ...` on this one as well?

Reply:
Yep!